AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -48,18 +48,54 @@ struct SessionHeaderView: View {
 
 private extension SessionHeaderView {
     var dateAndTime: some View {
-        guard let start = session.startTime else {
-            return Text("")
-        }
-        let end = session.endTime ?? Date()
         
         let formatter = DateIntervalFormatter()
-        
         formatter.timeStyle = .short
         formatter.dateStyle = .medium
-        let string = DateIntervalFormatter().string(from: start, to: end)
-        let replaced = string.replacingOccurrences(of: "—", with: "-")
-        return Text(replaced)
+        var date = ""
+        var additionalDate = ""
+        
+            guard let start = session.startTime else { return Text("") }
+            let end = session.endTime ?? Date()
+            let string = DateIntervalFormatter().string(from: start, to: end)
```

Comment:
I know you're all tired of me saying this, but we should split responsibilities and this should be handled by the viewmodel. Common interpretation for "single responsibility" is that an object should only have one reason to change. For the case of views it should solely be UI layout. Not formatting, not colors, not strings, just a screen layout.

As for now I think we should just roll with it as deadline is approaching, but in the future please keep that in mind and introduce proper viewmodels so that any data formatting decisions are being made there, not in the view itself.

Reply:
I will then. Haven't thought about viewmodel really, but it makes sense for sure after your explanation